kf_lib_data_ingest/common/constants.py

```diff
@@ -26,49 +28,75 @@ class HASH:
 
 class SPECIMEN:
     class COMPOSITION:
-        XENOGRAFT = 'Xenograft Tissue'
-        EBVI = 'Epstein-Barr Virus Immortalized Cells'
-        TISSUE = 'Solid Tissue'
-        BLOOD = 'Blood'
-        PERIPHERAL_BLOOD = 'Peripheral Blood'
-        BUCCAL_SWAB = 'Buccal Cells'
+        BLOOD = 'Peripheral Whole Blood'
+        BONE = 'Bone'
         BONE_MARROW = 'Bone Marrow'
+        BUCCAL_SWAB = 'Buccal Cells'
+        EBVI = 'Epstein-Barr Virus Immortalized Cells'
+        FIBROBLASTS = 'Fibroblasts'
+        LINE = 'Derived Cell Line'
+        LYMPHOCYTES = 'Lymphocytes'
+        MNC = 'Mononuclear Cells'
+        PLASMA = 'Plasma'
         SALIVA = 'Saliva'
-        LINES = 'Cell Lines'
-        BONE = "Bone"
+        TISSUE = 'Solid Tissue'
+        XENOGRAFT = 'Xenograft Tissue'
 
-    class TISSUE_TYPE:
-        TUMOR = 'Tumor'
-        GERMLINE = 'Normal'
+    class SAMPLE_PROCUREMENT:
+        AUTOPSY = 'Autopsy'
+        BIOPSY = 'Biopsy'
+        BLOOD_DRAW = 'Blood Draw'
+        SUBTOTAL_RESECTIONS = 'Subtotal Resections'
```

Comment:
Why is this plural?

Reply:
I believe these enums were added against GDC's data dictionary. On our API doc, subtotal resection and gross total resection are plural (https://kf-api-dataservice.kidsfirstdrc.org/#tag/Biospecimen%2Fpaths%2F~1biospecimens%2Fpost), but on GDC, they are singular (https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=sample).

Reply:
Hmm. Singular would be more grammatically consistent.

Reply:
Updated!